### DISCUSSION TITLE
NyatworkSSL网站无法登录

### DISCUSSION THREAD

**Discussion by wkkkkkk-kkk** [natfrp/wiki#214] [使用问题] [answered]
![image](https://user-images.githubusercontent.com/105221812/202633764-cd6a7ff1-e04b-4d1c-8c98-e13d529ff790.png)
NyatworkSSL网站无法登录
之前在上面买过几张证书，现在需要怎么办呜呜呜

**Comment by fakeboboliu** [comment, marked as answer]
您好，
正确的安全操作应当下载证书后自行妥善保存，我们并无保管义务。
如果需要找回的话请使用注册邮箱联系 support@natfrp.cloud ，由相关人员帮您找回证书。
